addon/components/lazy-render-wrapper.js

```diff
@@ -102,7 +103,7 @@ export default Ember.Component.extend({
 
 			return true;
 
-		} else if (this.get('isShown') || this.get('tooltipIsVisible')) {
+		} else if (!this.get('disabled') && (this.get('isShown') || this.get('tooltipIsVisible'))) {
 
 			this.set('_hasRendered', true);
 			return true;
```

Comment:
should I just handle `disabled` once before all of these conditionals, and `return false`?